authority/provisioner/webhook.go

```diff
@@ -141,7 +142,11 @@ type Webhook struct {
 	} `json:"-"`
 }
 
-func (w *Webhook) DoWithContext(ctx context.Context, client *http.Client, reqBody *webhook.RequestBody, data any) (*webhook.ResponseBody, error) {
+// TransportWrapper wraps the set of functions mapping [http.Transport] references to
+// [http.RoundTripper].
+type TransportWrapper = httptransport.Wrapper
```

Comment:
What is this for?

Reply:
The `httptransport` package is an internal one, to avoid duplication from package to package. Since the function's signature is inside function definitions I thought to compress the signature by declaring a type alias.